### DISCUSSION TITLE
get the db schema from the client

### DISCUSSION THREAD

**Discussion by danielzzz** [prisma/prisma#5406] [Q&A]
Hello, I would like to get the db schema from the client.
is there an easy way to do so?

I've seen there is an _dmmf object, but there is just too many properties there..

thanks, dan

**Comment by ryands17** [comment]
@danielzzz 👋 

Currently the only way to programmatically get the schema would be `_dmmf`. You can get the models via:

```ts
// @ts-ignore
console.log(prisma._dmmf.datamodel)
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
